src/main/java/com/newts/newtapp/api/application/RequestField.java

```diff
@@ -15,5 +15,7 @@
     TOPIC,
     USER_ID,
     USER_ID_TWO,
+    USER_KEY,
```

Comment:
WIth new implmementation you can get rid of these additions I believe. You may need other additions though.